pkg/3scale/amp/operator/redis_reconciler.go

```diff
@@ -124,3 +147,100 @@ func Redis(apimanager *appsv1alpha1.APIManager, client client.Client) (*componen
 	}
 	return component.NewRedis(opts), nil
 }
+
+func (r *RedisReconciler) deleteSystemRedisSecretNamespaceAttribute() (reconcile.Result, error) {
+	redis, err := Redis(r.apiManager, r.Client())
```

Comment:
No harm in this but I don't think we need it. We just want the access to secret.

Reply:
actually, that Redis is already initialized on line 58

Reply:
removed. Thank you